src/Http/Http.Extensions/src/ContentTypeConstants.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Http
+{
+    internal static class ContentTypeConstants
+    {
+        public const string PlainTextContentTypeWithCharset = "text/plain; charset=utf-8";
```

Comment:
You can just make this private and put it directly in RequestDelegateFactory. We should probably write the test with the string literal anyway, because if we change this string, it will be a behavioral change that should cause tests to fail imo.